app/scripts/models/oauth-app.js

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * OAuthApp information
+ */
+define(function (require, exports, module) {
+  'use strict';
+
+  var Backbone = require('backbone');
+  var Constants = require('lib/constants');
+
+  module.exports = Backbone.Model.extend({
```

Comment:
Wonder if there is enough overlap to have this extend from devices model or vice versa?